image/types/src/lib.rs

```diff
@@ -50,6 +50,7 @@ pub type ImageEccPrivKey = ImageScalar;
 
 #[repr(C)]
 #[derive(AsBytes, FromBytes, Default, Debug, Copy, Clone, Eq, PartialEq)]
+#[cfg_attr(feature = "arbitrary", derive(arbitrary::Arbitrary))]
```

Comment:
Maybe this is a dumb question, but doesn't the `arbitrary` feature need to be defined in image/types/Cargo.toml?

Reply:
As an optional dependency, the feature is automatically defined.

Reply:
Ack, thanks for clarifying.